src/embind/embind.js

```diff
@@ -917,51 +917,196 @@ var LibraryEmbind = {
+          setup(this);
+          prepareArg(0, arg0); prepareArg(1, arg1); prepareArg(2, arg2);
+          prepareArg(3, arg3); prepareArg(4, arg4); prepareArg(5, arg5);
+          var rv = cppInvokerFunc.apply(null, invokerFuncArgs);
+          cleanup();
+          return returns ? argTypes[0].fromWireType(rv) : undefined;
+        };
+      case 7:
+        return function(arg0, arg1, arg2, arg3, arg4, arg5, arg6) {
+          setup(this);
+          prepareArg(0, arg0); prepareArg(1, arg1); prepareArg(2, arg2);
+          prepareArg(3, arg3); prepareArg(4, arg4); prepareArg(5, arg5);
+          prepareArg(6, arg6);
+          var rv = cppInvokerFunc.apply(null, invokerFuncArgs);
+          cleanup();
+          return returns ? argTypes[0].fromWireType(rv) : undefined;
+        };
+      case 8:
+        return function(arg0, arg1, arg2, arg3, arg4, arg5, arg6, arg7) {
+          setup(this);
+          prepareArg(0, arg0); prepareArg(1, arg1); prepareArg(2, arg2);
+          prepareArg(3, arg3); prepareArg(4, arg4); prepareArg(5, arg5);
+          prepareArg(6, arg6); prepareArg(7, arg7);
+          var rv = cppInvokerFunc.apply(null, invokerFuncArgs);
+          cleanup();
+          return returns ? argTypes[0].fromWireType(rv) : undefined;
+        };
+      case 9:
+        return function(arg0, arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8) {
+          setup(this);
+          prepareArg(0, arg0); prepareArg(1, arg1); prepareArg(2, arg2);
+          prepareArg(3, arg3); prepareArg(4, arg4); prepareArg(5, arg5);
+          prepareArg(6, arg6); prepareArg(7, arg7); prepareArg(8, arg7);
+          var rv = cppInvokerFunc.apply(null, invokerFuncArgs);
+          cleanup();
+          return returns ? argTypes[0].fromWireType(rv) : undefined;
+        };
+      case 10:
+        return function(arg0, arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8,
+                        arg9) {
+          setup(this);
+          prepareArg(0, arg0); prepareArg(1, arg1); prepareArg(2, arg2);
+          prepareArg(3, arg3); prepareArg(4, arg4); prepareArg(5, arg5);
+          prepareArg(6, arg6); prepareArg(7, arg7); prepareArg(8, arg8);
+          prepareArg(9, arg9);
+          var rv = cppInvokerFunc.apply(null, invokerFuncArgs);
+          cleanup();
+          return returns ? argTypes[0].fromWireType(rv) : undefined;
+        };
+      case 11:
+        return function(arg0, arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8,
+                        arg9, arg10) {
+          setup(this);
+          prepareArg(0, arg0); prepareArg(1, arg1); prepareArg(2, arg2);
+          prepareArg(3, arg3); prepareArg(4, arg4); prepareArg(5, arg5);
+          prepareArg(6, arg6); prepareArg(7, arg7); prepareArg(8, arg8);
+          prepareArg(9, arg9); prepareArg(10, arg10);
+          var rv = cppInvokerFunc.apply(null, invokerFuncArgs);
+          cleanup();
+          return returns ? argTypes[0].fromWireType(rv) : undefined;
+        };
+      case 12:
```

Comment:
Why 12 here? Why 14 later? These seem pretty arbitrary.

Reply:
They are arbitrary; they're based on the largest number of arguments in our own application. Happy to add more, though.